src/main/java/uk/gov/digital/ho/hocs/audit/export/caseworkclient/dto/GetCorrespondentWithPrimaryFlagResponse.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.digital.ho.hocs.audit.export.caseworkclient.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public class GetCorrespondentWithPrimaryFlagResponse {
```

Comment:
Could the primary flag not be added to the existing GetCorrespondentOutlineResponse DTO?

Reply:
Yep that's a good plan, I've updated to do this instead